.travis.yml

```diff
@@ -48,3 +48,15 @@ addons:
     build_command_prepend: "./gradlew clean"
     build_command: "./gradlew build"
     branch_pattern: master
+
+deploy:
+  skip_cleanup: true
+  provider: releases
+  api_key:
+    secure: hWL8k4eTm6RBA6JtY+zL/bFFm+f2eM466kKlHBrywukSXAThgX8VzHVu9I8u9bL1yP2j8R9XHmG+C49Fy8NZ85Hmj5ECFg3dGRhfxAzr3o3deP3Kaeo+LozGhOy/z2PvOPyHS7XqOK8kk8gmoMJ9jvhz+wFAbW9GxhKLYBMn81E=
```

Comment:
everyone with write access is able to deploy releases, but the plan is, that only travis will do it.
With this api-token I've given travis access to this repo in order to deploy releases.

If @He3556 wants a token too, he can generate himself one here: https://github.com/settings/tokens/new (for travis the public_repo scope is needed)